internal/test/wsu/wsu_test.go

```diff
@@ -87,7 +87,11 @@ func TestWSU(t *testing.T) {
 	require.NotEmptyf(t, clusterAddress, "CLUSTER_ADDR environment variable not set")
 
 	for _, vm := range framework.WinVMs {
+		// Run the test suite twice, to ensure that the WSU can be run multiple times against the same VM
 		runWSUTestSuite(t, vm)
+		t.Run("Run the WSU against the same VM again", func(t *testing.T) {
```

Comment:
Why are you doing it against every vm? Won't it be sufficient, if we run against single VM once?

Reply:
Resiliency. If we can at little extra time cost, why not? Currently we are only spinning up one VM, and when we add multiple VMs in the future the tests can be run in parallel, see: #123

Reply:
To clarify, the seperate VM tests can be run in parallel, not the test i am adding here

Reply:
As discussed offline, I'm fine with this change.